GH95 | error is not returned when mic access is blocked (#144)
twilio/twilio-voice.js: CHANGELOG.md

2.3.2 (In Progress)
===================

Bug Fixes
---------

> - Fixed an [issue](https://github.com/twilio/twilio-voice.js/issues/95) where Twilio error code is not being returned when microphone access is blocked. Thank you @SiimMardus and @ostap0207 for your [contribution](https://github.com/twilio/twilio-voice.js/pull/143).

Reviewer: ```suggestion
- Fixed an [issue](https://github.com/twilio/twilio-voice.js/issues/95) where a Twilio Error is not returned when microphone access is blocked. Thank you @SiimMardus and @ostap0207 for your [contribution](https://github.com/twilio/twilio-voice.js/pull/143).
```
